.github/workflows/Deploy.yml

```diff
@@ -38,7 +38,7 @@ jobs:
             -   name: Set up PHP
                 uses: shivammathur/setup-php@v2
                 with:
-                    php-version: 8.3
```

Comment:
did you verify tailor is PHP 8.4 compatible?

Reply:
According to https://github.com/TYPO3/tailor/blob/a86b7c67779eec0ea743fb27f9f5f597137ed7fe/composer.json#L21, PHP 8.4 support is given.

I was at least able to create the extension artifact locally:

![image](https://github.com/user-attachments/assets/cb14e82c-9c85-4223-b431-8ae2a3828a62)

Reply:
For completeness: Tailor is currently [not tested against PHP 8.4](https://github.com/TYPO3/tailor/blob/a86b7c67779eec0ea743fb27f9f5f597137ed7fe/.github/workflows/tests.yml#L9).

Reply:
Looks good here: https://github.com/TYPO3/tailor/actions/runs/11569385322/job/32203001241?pr=79

Reply:
Dude, you're on fire. :fire: Thanks for double-checking!